README.md

```diff
@@ -799,6 +803,80 @@ If you are using a feature flag with binary variants, the `StatusOverride` prope
 
 In the above example, the feature is enabled by the `AlwaysOn` filter. If the current user is in the calculated percentile range of 10 to 20, then the `On` variant is returned. Otherwise, the `Off` variant is returned and because `StatusOverride` is equal to `Disabled`, the feature will now be considered disabled.
 
+## Telemetry
+
+When a feature flag change is deployed, it is often important to analyze its effect on an application. For example, here are a few questions that may arise:
+
+* Are my flags enabled/disabled as expected?
+* Are targeted users getting access to a certain feature as expected?
+* What variant is a particular user seeing?

```

Comment:
NIT: Which variant...

The variants are predefined, so it's more about which variant instead of what variant.